In [1]:
import requests
import pandas as pd
import numpy as np

In [13]:
!ls

Project Fletcher Proposal.pdf reddit-ETL.ipynb
README.md                     results.csv
Untitled.ipynb                seinfeld-chronicles
Untitled1.ipynb               showerthoughts.csv
darkweb-EDA.ipynb             test_csv.csv.


In [14]:
df = pd.read_csv('showerthoughts.csv')

In [15]:
df.columns.values

array(['author', 'title', 'subreddit', 'score', 'num_comments',
       'retrieved_on', 'id', 'created_utc', 'selftext'], dtype=object)

In [16]:
len(df[df.score > 1]) / len(df)

0.392702

39% of submissions get more than one upvote. This makes "greater than one upvote" a good candidate for our criteria for a good post.